mne/gui/tests/test_ieeg_locate_gui.py

```diff
@@ -0,0 +1,136 @@
+              (x - 20, y - 16, z + 1), (x - 30, y - 16, z + 1)]
+    center = np.array(brain.shape) / 2
+    # make image
+    np.random.seed(99)
+    ct_data = np.random.random(brain.shape).astype(np.float32) * 100
+    # make skull
+    for vert in verts:
+        x, y, z = np.round(vert).astype(int)
+        ct_data[slice(x - skull_size, x + skull_size + 1),
+                slice(y - skull_size, y + skull_size + 1),
+                slice(z - skull_size, z + skull_size + 1)] = 1000
+    # add electrode with contacts
+    for (x, y, z) in coords:
+        # make sure not in skull
+        assert np.linalg.norm(center - np.array((x, y, z))) < 50
+        ct_data[slice(x - contact_size, x + contact_size + 1),
+                slice(y - contact_size, y + contact_size + 1),
+                slice(z - contact_size, z + contact_size + 1)] = \
+            1000 - np.linalg.norm(np.array(np.meshgrid(
+                *[range(-contact_size, contact_size + 1)] * 3)), axis=0)
+    ct = nib.MGHImage(ct_data, brain.affine)
+    coords = mne.transforms.apply_trans(
+        ct.header.get_vox2ras_tkr(), np.array(coords))
+    return ct, coords
+
+
+@requires_nibabel()
+def test_ieeg_elec_locate_gui_io():
+    """Test the input/output of the intracranial location GUI."""
+    import nibabel as nib
+    info = mne.create_info([], 1000)
+    aligned_ct = nib.MGHImage(np.zeros((256, 256, 256), dtype=np.float32),
+                              np.eye(4))
+    trans = mne.transforms.Transform('head', 'mri')
+    with pytest.raises(ValueError,
+                       match='No channels found in `info` to locate'):
+        mne.gui.locate_ieeg(info, aligned_ct, subject, subjects_dir)
+    info = mne.create_info(['test'], 1000, ['seeg'])
+    with pytest.raises(ValueError, match='CT is not aligned to MRI'):
+        mne.gui.locate_ieeg(info, trans, aligned_ct, subject=subject,
+                            subjects_dir=subjects_dir)
+
+
+@requires_version('pyvista')
+@testing.requires_testing_data
+def test_ieeg_elec_locate_gui_display():
+    """Test that the intracranial location GUI displays properly."""
+    raw = mne.io.read_raw_fif(raw_path)
+    raw.pick_types(eeg=True)
+    ch_dict = {'EEG 001': 'LAMY 1', 'EEG 002': 'LAMY 2',
+               'EEG 003': 'LSTN 1', 'EEG 004': 'LSTN 2'}
+    raw.pick_channels(list(ch_dict.keys()))
+    raw.rename_channels(ch_dict)
+    raw.set_montage(None)
+    aligned_ct, coords = _make_fake_CT(subject, subjects_dir)
+    trans = mne.read_trans(fname_trans)
+    with pytest.warns(RuntimeWarning, match='`pial` surface not found'):
+        gui = mne.gui.locate_ieeg(raw.info, trans, aligned_ct,
+                                  subject=subject, subjects_dir=subjects_dir)
+
+    gui._ras[:] = coords[0]  # start in the right position
+    gui._move_cursors_to_pos()
+    for coord in coords:
+        coord_vox = mne.transforms.apply_trans(gui._ras_vox_t, coord)
+        _fake_click(gui._figs[2], gui.figs[2].axes[0],
```

Comment:
From the logs:
```
mne\gui\tests\test_ieeg_locate_gui.py:100: in test_ieeg_elec_locate_gui_display
    _fake_click(gui._figs[2], gui.figs[2].axes[0],
E   AttributeError: 'IntracranialElectrodeLocator' object has no attribute 'figs'
```
It's possible the PyVista errors all just come from this (?), so I'll fix it here and hopefully that fixes CIs.
```suggestion
        _fake_click(gui._figs[2], gui._figs[2].axes[0],
```